config/locales/proper.en.yml

```diff
@@ -242,9 +254,11 @@ en:
         get_cost_estimate: "Get a Cost Estimate"
         save_and_continue: "Save and Continue →"
         submit: "Submit Request →"
+        submit_request: "Submit Request"
         saving: "Saving..."
         submitting: "Submitting..."
         getting_cost: "Getting Cost..."
+        view_study_details: "View Study Details"
```

Comment:
This value is pulled from `t('protocols.view_details.button')` in the application and needs a `protocol_type` parameter.

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/protocols/_summary.html.haml
https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/dashboard/protocols/_summary.html.haml